helmclient.go

```diff
@@ -541,6 +541,24 @@ func (c *Client) ListReleaseContents(ctx context.Context) ([]*ReleaseContent, er
 	return contents, nil
 }
 
+// LoadChart loads a Helm Chart and returns relevant parts of its structure.
+func (c *Client) LoadChart(ctx context.Context, chartPath string) (*Chart, error) {
```

Comment:
I think it should return a value. This is just a data type.

```suggestion
func (c *Client) LoadChart(ctx context.Context, chartPath string) (Chart, error) {
```

Reply:
OK makes sense. Done.